packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -179,14 +178,11 @@ export const BookEventForm = ({ onCancel }: BookEventFormProps) => {
     return defaults;
   }, [eventType?.bookingFields, formValues, isRescheduling, bookingData, rescheduleUid]);
 
-  const disableBookingTitle = !!event.data?.isDynamic;
   const bookingFormSchema = z
     .object({
       responses: event?.data
         ? getBookingResponsesSchema({
-            eventType: {
-              bookingFields: getBookingFieldsWithSystemFields({ ...event.data, disableBookingTitle }),
-            },
+            eventType: event?.data,
```

Comment:
Its already gone through `getBookingFieldsWithSystemFields` in SSR. This was already handled in old booker but not in New booker